### ISSUE TITLE
Make stats with subprojet

### ISSUE THREAD

**Issue by skalimer0** [fraoustin/redmine_indicator#14]
Hello,

Very great job, but if we can make indicator on projets with sub-project issues (by option to check) it's will be very useful.

thanks

**Comment by sebd71** [comment]
Yes, I'm also very interested to have project(s) and/or sub-projects metrics.

**Comment by fraoustin** [comment]
Hi 

you can display the indicator in 

- your page (all projects) : https://url redmine/my/page
- for each project page (one project): https://url redmine/projects/name project
You want add in your page the indicator by project ?

**Comment by sebd71** [comment]
No my goal is to have global (no specifically assigned to me) metrics per project.
For example a graph showing the number of open / closed issues for a specific project.
I didn't find where you build filters to retrieve data, but from what I see, you add "assignee is me" in filter, something like :
  Status is "open" AND Assignee is "me"
and my need is more a filter like : 
  Status is "open" AND Project is "xyz"

**Comment by fraoustin** [comment]
I have not feature of " to manage metric with add specific filter by user" ... my idea is to offer a standard metric for basic user

**Comment by skalimer0** [comment]
This issue is just to put in indicator's charts issuse from the projet and issues from his subprojets.

For example we have subprojets for software's modules (core / webapp / webservice / ...)
Same person work on these subprojets and we want indicator on the main projet with all issues.

**Comment by skalimer0** [comment]
It's the line :

query.add_filter "project_id", "=", [project.id]

to make optional or create another filter without this line to have subprojet issues

**Comment by skalimer0** [comment]
To have subproject issues...

In "app/views/projects/blocks" select the query to modify
replace the line : query.add_filter "project_id", "=", [project.id]
by this line : query.project = project

Done.

**Comment by CSCSxx** [comment]
I would like this feature too!
In the project settings there should be an option "Include issues from subprojects"
